arches/app/views/graph.py

```diff
@@ -195,15 +193,24 @@ def get(self, request, graphid):
         if graphid == settings.SYSTEM_SETTINGS_RESOURCE_MODEL_ID:
             if not request.user.groups.filter(name="System Administrator").exists():
                 raise PermissionDenied
-            
+
         self.source_graph = Graph.objects.get(pk=graphid)
+        if self.source_graph.source_identifier_id:
+            url = reverse('graph_designer', kwargs={'graphid': self.source_graph.source_identifier_id})
+            
+            query_string = urlencode({
+                'has_been_redirected_from_editable_future_graph': True
+            })
```

Comment:
In case we (or an implementer) add more query params, this should prevent overwriting any others:
```suggestion
            query_dict = request.GET.copy()
            query_dict["has_been_redirected_from_editable_future_graph"] = True
            query_string = query_dict.urlencode()
```